util/audio.py

```diff
@@ -2,9 +2,10 @@
 
 import scipy.io.wavfile as wav
 import sys
+import math
 
 try:
-    from deepspeech.utils import audioToInputVector
+    from deepspeech import audioToInputVector
 except ImportError:
     import numpy as np
     from python_speech_features import mfcc
```

Comment:
nit: we should update the warning message, to make it more clear:
```
DeepSpeech Python bindings could not be loaded, resorting to slower code to compute the audio vectors.
Refer to README.md for instructions on how to install (or build) deepspeech python bindings.
```

Reply:
Good catch, will do.